OptimizelySDK/Odp/Constants.cs

```diff
@@ -65,7 +65,12 @@ public static class Constants
         /// <summary>
         /// Default message to log when an ODP Event contains invalid data
         /// </summary>
-        public const string ODP_INVALID_DATA_MESSAGE = "ODP data is not valid.";
+        public const string ODP_INVALID_DATA_MESSAGE = "ODP event send failed (event data is not valid).";
+
+        /// <summary>
+        /// Default message to log when an ODP Identifiers is invalid
+        /// </summary>
+        public const string ODP_INVALID_IDENTIFIERS_MESSAGE = "ODP event send failed (event identifiers must have at least one key-value pair).";
```

Comment:
I'm open to adding new (more descriptive) messages as long as @jaeopt 's cool with it.